<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWindow</class>
 <widget class="QDialog" name="LoginWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>404</width>
    <height>510</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* =========================
   MAIN WINDOW
   ========================= */
QDialog {
    background-color: #121212;
    color: white;
    font-family: Segoe UI;
    font-size: 13px;
}

/* =========================
   LOGIN CARD (ORANGE BORDER)
   ========================= */
QFrame {
    background-color: transparent;
    padding: 16px;
}

/* =========================
   TEXT
   ========================= */
QLabel {
    color: #E0E0E0;
}

QLabel#lblTitle {
    color: #FF9800;
    font-size: 24px;
    font-weight: bold;
}

/* =========================
   INPUT FIELDS (BORDER STYLE)
   ========================= */
QLineEdit {
    background-color: transparent;
    color: white;
    border: 1.5px solid #FF9800;
    border-radius: 8px;
    padding: 8px;
}

QLineEdit:focus {
    border: 2px solid #FFA726;
}

/* =========================
   LOGIN BUTTON (OUTLINE STYLE)
   ========================= */
QPushButton {
    background-color: transparent;
    color: #FF9800;
    font-weight: bold;
    border: 2px solid #FF9800;
    border-radius: 10px;
    padding: 10px;
}

QPushButton:hover {
    background-color: rgba(255, 152, 0, 25);
}

QPushButton:pressed {
    background-color: rgba(255, 152, 0, 60);
    color: black;
}

/* =========================
   DISABLED BUTTON
   ========================= */
QPushButton:disabled {
    border-color: #555;
    color: #555;
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,0,3,1">
     <property name="spacing">
      <number>12</number>
     </property>
     <property name="leftMargin">
      <number>30</number>
     </property>
     <property name="topMargin">
      <number>30</number>
     </property>
     <property name="rightMargin">
      <number>30</number>
     </property>
     <property name="bottomMargin">
      <number>30</number>
     </property>
     <item>
      <widget class="QFrame" name="frameLogin">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>120</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="lblTitle">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>65</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="sizeIncrement">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Segoe UI Emoji</family>
            <pointsize>-1</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Welcome Back</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lbltop">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>60</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Pleasse sign in continue</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QWidget" name="widget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>1</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>326</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>326</width>
         <height>200</height>
        </size>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="lblUsername">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>45</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>35</height>
           </size>
          </property>
          <property name="text">
           <string>Username</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="leUsername">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>35</height>
           </size>
          </property>
          <property name="placeholderText">
           <string>Enter your username</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblPassword">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>45</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>35</height>
           </size>
          </property>
          <property name="text">
           <string>Password</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lePassword">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>35</height>
           </size>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
          <property name="placeholderText">
           <string>Enter your Password</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnLogin">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>-1</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: #FF9800;
    color: black;
    font-size: 14px;
    font-weight: bold;
    border-radius: 10px;
    padding: 10px;
    border: none;
}

QPushButton:hover {
    background-color: #FFA726;
}

QPushButton:pressed {
    background-color: #F57C00;
}
</string>
       </property>
       <property name="text">
        <string>LOGIN</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
